src/Sarif/PropertiesDictionaryExtensionMethods.cs

```diff
@@ -95,6 +95,20 @@ public static class PropertiesDictionaryExtensionMethods
                     continue;
                 }
 
+                FailureLevelSet failureLevelSet = property as FailureLevelSet;
```

Comment:
>failureLevelSet [](http://example.com/codeflow?start=32&length=15)

changes in this file allow for clean serialization and deserialization of sets that contain failure levels and result kinds, highly used to configure analysis.